### PR TITLE
Sync open source GFR with Stadia

### PR DESCRIPTION
**Pull request by zakerinasab**
This pull syncs the open source GFR with the changes made
in the Stadia GFR.

### CODE REVIEW

File: generated/gfr.json.inc [hunk 1]
```diff
@@ -3,10 +3,10 @@
   "layer": {
     "name": "VK_LAYER_GOOGLE_graphics_flight_recorder",
     "type": "GLOBAL",
-    "library_path": "libVkLayer_gfr.so",
+    "library_path": "/usr/local/cloudcast/lib/libVkLayer_gfr.so",
```
Comment (aellem) [inline generated/gfr.json.inc]: I purposely removed the path

File: generated/gfr.json.inc [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
     "api_version": "1.1.73",
     "implementation_version": "1",
-    "description": "Graphics Flight Recorder is a crash/hang debugging tool that helps determines GPU progress in a Vulkan application",
+    "description": "Google Graphics Flight Recorder is a crash/hang debugging tool that helps determines GPU progress in a Vulkan application",
```
Comment (aellem) [inline generated/gfr.json.inc]: Same here (we should fix the internal name).  As this isn't an official Google product we shouldn't use Google in the name.

**Review by aellem — CHANGES_REQUESTED** [review]

File: generated/gfr.json.inc (hunk 1 above)
Comment (zakerinasab) [reply generated/gfr.json.inc]: Done.

File: generated/gfr.json.inc (hunk 2 above)
Comment (zakerinasab) [reply generated/gfr.json.inc]: Done.

**Review by aellem — APPROVED** [review]

**Pull request merged**